elasticgraph-schema_artifacts/lib/elastic_graph/schema_artifacts/runtime_metadata/scalar_type.rb

```diff
@@ -14,7 +14,8 @@ module RuntimeMetadata
       # Provides runtime metadata related to scalar types.
       #
       # @private
-      class ScalarType < ::Data.define(:coercion_adapter_ref, :indexing_preparer_ref)
+      class ScalarType < ::Data.define(:coercion_adapter_ref, :indexing_preparer_ref, :grouping_missing_value_placeholder)
+
```

Comment:
```suggestion
```

This is causing a build failure. (I'd encourage you to run `script/quick_build` before pushing at it surfaces this stuff pretty quickly).